### PR TITLE
Bulk Keys support in ssv-keys

### PR DESCRIPTION
**Pull request by vadiminc**
Fixes #<issue_number_goes_here>

> It's a good idea to open an issue first for discussion.

- [ ] Tests pass
- [ ] Appropriate changes to README are included in PR

### CODE REVIEW

File: src/cli-shared.ts [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import figlet from 'figlet';
+import colors from 'colors/safe';
 import pkg from '../package.json';
+
```
Comment (axelrod-blox) [inline src/cli-shared.ts:4]: unnecessary space

File: src/commands/actions/KeySharesAction.ts [hunk 2]
```diff
@@ -12,7 +11,7 @@ import outputFolderArgument from './arguments/output-folder';
 import operatorPublicKeysArgument from './arguments/operator-public-keys';
 import { keystorePasswordValidator } from './validators/keystore-password';
```
Comment (axelrod-blox) [inline src/commands/actions/KeySharesAction.ts:13]: empty field

File: src/commands/actions/arguments/password.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { keystorePasswordValidator } from '../validators/keystore-password';
+// import { keystorePasswordValidator } from '../validators/keystore-password';
```
Comment (axelrod-blox) [inline src/commands/actions/arguments/password.ts:1]: If we doesn't need it let's remove

**Review by axelrod-blox — APPROVED** [review]

**Review by meshin-dev — CHANGES_REQUESTED** [review]
@vadiminc
1. Jon have refactored PRD, we should listen to him and revise what's done
2. I think this solution is over-complicated and we should take a closer look of how to use advantages of architecture and try to not change deeper levels of the code if not necessary

File: src/cli-shared.ts [hunk 4]
```diff
@@ -26,5 +28,11 @@ export default async function main(interactive: boolean): Promise<any> {
     console.log(' ----------------------------------------------------------------------\n');
   }
   const command = new SSVKeysCommand(interactive);
-  await command.execute().then(console.debug).catch(console.error);
+  try {
+    const outputFiles = await command.execute();
+    console.debug(`\nKey distribution successful! Find your key shares file${outputFiles.length > 1 ? 's': ''} at:`);
+    outputFiles.map((file: string) => console.debug(`${colors.bgYellow(colors.black(file))}`));
```
Comment (meshin-dev) [inline src/cli-shared.ts:34]: Use forEach instead of map)

**Review by meshin-dev — APPROVED** [review]

**Review by andrew-blox — APPROVED** [review]

**Pull request merged**